setup.py

```diff
@@ -90,7 +90,7 @@ def run(self):
 requirements = [
     "galaxy-importer==0.4.2",
     "pulpcore>=3.18.1,<3.19.0",
-    "pulp-ansible>=0.12.0,<0.13.0",
+    "pulp-ansible@git+https://github.com/pulp/pulp_ansible@main#egg=pulp-ansible",  # noqa E501
```

Comment:
if our commit needs to include a non-released pulp-ansible, this should be a commit hash rather than `main` so we can have reproducible builds

Reply:
@awcrosby I couldn't get a commit to work with the plugin template. Is it okay to set setup.py to a commit and the plugin template to main?

Reply:
if the `requirements/` files have the commit fed from setup.py, that should be good enough for reproducible builds. for now i think that is ok.